x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -583,6 +586,12 @@ export const ConfigurationStepForm: FC<ConfigurationStepProps> = ({
       <Fragment>
         <Messages messages={requestMessages} />
         <SupportedFieldsMessage jobType={jobType} />
+        {hasRequiredIndicesPermissions === false ? (
+          <>
+            <IndexPermissionsCallout indexName={sourceDataViewTitle} docsType="create" />
```

Comment:
`hasRequiredIndicesPermissions` could be moved inside the `IndexPermissionsCallout` component.

Reply:
Good call - moved in 51e4a289dfc2e1810a7a3ab1d28febae50c405cc